pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
It's just a cheat to make the call to map easily, since I'm iterating over tuples and multiprocessing.Pool doesn't have a mapstar method in python 2.7. Or does it?